markdown/ch3.markdown

```diff
@@ -8,7 +8,7 @@
 > — William Blake, *The Tyger*
 
 You probably know that digital images are made up of *pixels*. These are
-the light signal *sampled on a regular grid*. When computing
+*samples* of the light signal *on a regular grid*. When computing
```

Comment:
@stefanv what about the current version? Another alternative is to ignore this altogether, since we move on from it rather quickly. But it's a good opportunity to educate... =\